php_phongo.c

```diff
@@ -2799,6 +2922,13 @@ void phongo_manager_init(php_phongo_manager_t* manager, const char* uri_string,
 	}
 #endif
 
+#ifdef MONGOC_ENABLE_CLIENT_SIDE_ENCRYPTION
```

Comment:
Did we previously discuss throwing if `autoEncryption` was specified but FLE was not enabled? I believe libmongoc has logic in `mongoc_client_enable_auto_encryption()` to raise an error in that case, but perhaps that wasn't implemented when you first drafted this PR. Assuming `mongoc_client_enable_auto_encryption()` is always available, perhaps we just need to remove the `#ifdef` checks here and around `phongo_manager_set_auto_encryption_opts`.

I've acknowledged your earlier comment that we don't have an environment to actually test for such an error, since libmongocrypt is always compiled in.

Reply:
Now that you mention it, we did indeed discuss throwing an exception in this case, since quietly disabling FLE might pose a security risk. While libmongoc would fail gracefully, the resulting error message could be confusing for users:
> libmongoc is not built with support for Client-Side Field Level Encryption. Configure with ENABLE_CLIENT_SIDE_ENCRYPTION=ON.

Given that, we may opt to throw our own error to inform users that they need to compile with the `mongodb-client-side-encryption` flag.